django_tutorial/first_project/models.py

```diff
@@ -0,0 +1,436 @@
+    last_name = models.CharField(max_length=30)
+    email = models.CharField(max_length=254)
+    is_staff = models.BooleanField()
+    is_active = models.BooleanField()
+    date_joined = models.DateTimeField()
+
+    class Meta:
+        managed = False
+        db_table = 'auth_user'
+
+
+class AuthUserGroups(models.Model):
+    user = models.ForeignKey(AuthUser, models.DO_NOTHING)
+    group = models.ForeignKey(AuthGroup, models.DO_NOTHING)
+
+    class Meta:
+        managed = False
+        db_table = 'auth_user_groups'
+        unique_together = (('user', 'group'),)
+
+
+class AuthUserUserPermissions(models.Model):
+    user = models.ForeignKey(AuthUser, models.DO_NOTHING)
+    permission = models.ForeignKey(AuthPermission, models.DO_NOTHING)
+
+    class Meta:
+        managed = False
+        db_table = 'auth_user_user_permissions'
+        unique_together = (('user', 'permission'),)
+
+
+class sba_zip_level(models.Model):
+    borr_zip = models.TextField(blank=True, null=True)
+    mean_agi = models.FloatField(blank=True, null=True)
+    total_small_bus = models.BigIntegerField(blank=True, null=True)
+    total_sba = models.BigIntegerField(blank=True, null=True)
+    total_504 = models.BigIntegerField(blank=True, null=True)
+    total_7a = models.BigIntegerField(blank=True, null=True)
+    sba_per_small_bus = models.FloatField(blank=True, null=True)
+    loan_504_per_small_bus = models.FloatField(blank=True, null=True)
+    loan_7a_per_small_bus = models.FloatField(blank=True, null=True)
+
+    class Meta:
+        managed = False
+        db_table = 'sba_zip_level'
+
+
+class DjangoAdminLog(models.Model):
+    action_time = models.DateTimeField()
+    object_id = models.TextField(blank=True, null=True)
+    object_repr = models.CharField(max_length=200)
+    action_flag = models.SmallIntegerField()
+    change_message = models.TextField()
+    content_type = models.ForeignKey('DjangoContentType', models.DO_NOTHING, blank=True, null=True)
+    user = models.ForeignKey(AuthUser, models.DO_NOTHING)
+
+    class Meta:
+        managed = False
+        db_table = 'django_admin_log'
+
+
+class DjangoContentType(models.Model):
+    app_label = models.CharField(max_length=100)
+    model = models.CharField(max_length=100)
+
+    class Meta:
+        managed = False
+        db_table = 'django_content_type'
+        unique_together = (('app_label', 'model'),)
+
+
+class DjangoMigrations(models.Model):
+    app = models.CharField(max_length=255)
+    name = models.CharField(max_length=255)
+    applied = models.DateTimeField()
+
+    class Meta:
+        managed = False
+        db_table = 'django_migrations'
+
+
+class DjangoSession(models.Model):
+    session_key = models.CharField(primary_key=True, max_length=40)
+    session_data = models.TextField()
+    expire_date = models.DateTimeField()
+
+    class Meta:
+        managed = False
+        db_table = 'django_session'
+
+
+class IrsZipData(models.Model):
```

Comment:
Do we need this model? Is best practice to just create model that we use in our API? Since our API (for now) is only using `trg_analytics.sba_region_level` should we just have one model for that table?